docs/pages/updates/index.tsx

```diff
@@ -167,7 +171,75 @@ export default function WhatsNew() {
           gap: 0,
         })}
       >
-        <Timeline date="6th September 2021" isLatest />
+        <Timeline date="21st September 2021" isLatest />
+        <Box heading="Keystone Community Q&A + YouTube launch ">
+          Taking the chance to introduce the core team and address some community questions in
+          person, the team held an online Q&A event and launched a Youtube channel at the same time.{' '}
+          <a
+            href="https://www.youtube.com/watch?v=r1IJh-iHm1c"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Watch it online here
+          </a>
+          .
+        </Box>
+        <Timeline date="16th September 2021" />
+        <Box heading="create-keystone-app now uses SQLite">
+          Our CLI app now uses <InlineCode>SQLite</InlineCode> under the hood so you don’t have to
+          spend time on DB config when trying out new ideas. We also updated the 
+          <Link href="/docs/walkthroughs/getting-started-with-create-keystone-app">
+            <a>getting started walkthrough</a>
+          </Link>{' '}
+          to reflect this improvement.
+        </Box>
+        <Timeline date="15th September 2021" />
+        <Box heading="Deployment examples for Heroku and Railway">
+          Learn how to get your Keystone project on the web with out new one-click starters for{' '}
+          <a
+            href="https://github.com/keystonejs/keystone-6-heroku-example"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Heroku
+          </a>{' '}
+          and{' '}
+          <a
+            href="https://github.com/keystonejs/keystone-6-railway-example"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Railway
+          </a>
+          .
+        </Box>
+        <Timeline date="9th September 2021" />
+        <Box heading="Prisma Korea Meetup">
+          Jed spoke at the Prisma Korea Meetup, covering V6 general availability, User-facing
```

Comment:
The capitalisation here seems a bit haphazard here, not quite sure what it should be though 🤔

Reply:
Totally, adjusted.